### ISSUE TITLE
More project ideas needed!

### ISSUE THREAD

**Issue by zvr** [spdx/GSoC#3]
random personal ideas:

-	Outreach: help with the website
-	Legal: help with license submission tools, help with bulk import from other license lists
-	Tech: help with SPDXv3 implementation in Java, Go, etc.

**Comment by jlovejoy** [comment]
have submitted something similar to your second bullet!!

**Comment by shreyanshhx** [comment]
Hey @jlovejoy @zvr 

Is there any slack channel or any other communication channel where all the potential GSoC'23 contributors can join?

**Comment by shreyanshhx** [comment]
Hey @zvr @jlovejoy 

Sorry this might sound silly, but is there any slack channel or any developer community channel where potential GSoC'23 contributors can join and discuss their ideas and queries with mentors?

**Comment by zvr** [comment]
@shreyanshhx we usually work on the Tech team mailing list -- and there is also a Gitter room. Check out https://spdx.dev/participate/tech/

**Comment by DiviyamPathak** [comment]
do you need some tests

**Comment by git-devansh** [comment]
Hi, I am interested in the topic _Fix Manifest Parsers for SPDX generator_. The open-source world is new to me, but I'd like to learn and contribute. How can I connect with the given mentor for that topic?

**Comment by ADITYADAS1999** [comment]
Hey, I am interested for gsoc this year, is there any gsoc project idea related docs present in repo ?

**Comment by zvr** [comment]
@ADITYADAS1999 yes, there is: https://github.com/spdx/GSoC/blob/main/2023_ideas.md

**Comment by RANJEETJ06** [comment]
i am a begginer for open source and please suggest me a issue to solve

**Comment by vivekd01** [comment]
Please give me some project to solve the problem.

**Comment by bact** [comment]
GSoC 2026 announced https://opensource.googleblog.com/2025/12/shape-future-with-google-summer-of-code.html
